CONTRIBUTING.md

```diff
@@ -5,134 +5,51 @@
 
 The following is a set of guidelines for contributing to this repository.
 
-
```

Comment:
GitHub automatically generates table of content for markdown files
![image](https://user-images.githubusercontent.com/26760571/190890908-2320c730-518e-47a1-a80b-a51006e55d4d.png)